pilot/lib/pilot/options.rb

```diff
@@ -22,7 +22,7 @@ def self.available_options
                                      description: "The bundle identifier of the app to upload or manage testers (optional)",
                                      optional: true,
                                      code_gen_sensitive: true,
-                                     default_value: ENV["TESTFLIGHT_APP_IDENTITIFER"] || CredentialsManager::AppfileConfig.try_fetch_value(:app_identifier),
+                                     default_value: ENV["PILOT_APP_IDENTIFIER"] || ENV["TESTFLIGHT_APP_IDENTITIFER"] || CredentialsManager::AppfileConfig.try_fetch_value(:app_identifier),
```

Comment:
Is this actually needed since `PILOT_APP_IDENTIFIER` is set as the `env_name` for this option? 🤔

Reply:
I don't think `try_fetch_value` gets called if the `PILOT_APP_IDENTIFIER` env is already set because there the `env_name` has a higher "default value" priority than `default_value` 😖 But I will test to verify my understandings are correct

Reply:
Docs show the ENV should have higher priority - https://docs.fastlane.tools/advanced/#priorities-of-parameters-and-options

Also, this shows if the `env_name` is defined and if that env actually has a value, that will get loaded first and the `default_value` won't get called - https://github.com/fastlane/fastlane/blob/master/fastlane_core/lib/fastlane_core/configuration/configuration.rb#L219-L227

I _think_ that the only reason the `_IDENTITIFER` env names are in the `default_value` are for backwards compatibility for a mistake that was made 😁

Reply:
Oh, so then this PR is invalid, right?

Reply:
(Please create a PR that documents via a comment why those ENV vars are there as a default_value then.)

Reply:
@janpio, That was my plan too 😛